lib/rototiller/task/params/env_var.rb

```diff
@@ -47,16 +47,16 @@ def message
         this_message = String.new
 
         if env_status    == STATUS[:nodefault_noexist]
-          this_message << red_text('ERROR: environment-variable not set and no default provided: ')
+          this_message << red_text(' - ERROR: environment-variable not set and no default provided: ')
```

Comment:
Just to give some separation between the message and the status message.  Right now it prints out like this:
`Beaker pre-suiteINFO: no system environment-variable value, using default provided: 'BEAKER_PRE_SUITE': '../../setup/high_availability/install.rb': ''`

This would make it print out like this: 
`Beaker pre-suite - INFO: no system environment-variable value, using default provided: 'BEAKER_PRE_SUITE': '../../setup/high_availability/install.rb': ''`

It could be a space if we just want that.